### ISSUE TITLE
🚀 Feature: PagerDuty cards in homepage

### ISSUE THREAD

**Issue by gaurav517** [backstage/backstage#15778]
### 🔖 Feature description

It would be good to have PagerDuty information in homepage. They can be can be static cards based on integration keys.

### 🎤 Context

We have two PagerDuty services that covers all our microservices. It would be good to see the oncall details at home page for both PD services.

### ✌️ Possible Implementation

One idea could be a more generalised version of `EntityPagerDutyCard` that takes integration key as input in props and shows the card.

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, but I'm happy to collaborate on a PR with someone else

**Comment by parifuture** [comment]
We already have a plugin for PageDuty, is it not what you are looking for? https://github.com/backstage/backstage/tree/master/plugins/pagerduty

**Comment by gaurav517** [comment]
I know about that plugin but I am not sure whether that is extensible enough to be used in homepage. (e.g. Can we pass title and service-id as props?). Of existing plugin is good enough to be used in home page, can we please get some instructions? Thanks.

**Comment by parifuture** [comment]
the module `@backstage/plugin-pagerduty` exports `EntityPagerDutyCard` which can be used anywhere (homepage included). And all the required instructions are in the readme file. have you tried exporting and importing the card in your `home.tsx` file?

**Comment by gaurav517** [comment]
I tried using it in homepage, it gives following error:
```
Error
Entity context is not available
Call Stack
 useEntity
  node_modules/@backstage/plugin-catalog-react/dist/index.esm.js:622:11
 PagerDutyCard
  node_modules/@backstage/plugin-pagerduty/dist/index.esm.js:631:96
 renderWithHooks
  node_modules/react-dom/cjs/react-dom.development.js:14985:18
 updateFunctionComponent
  node_modules/react-dom/cjs/react-dom.development.js:17356:20
 mountLazyComponent
  node_modules/react-dom/cjs/react-dom.development.js:17677:17
 beginWork
  node_modules/react-dom/cjs/react-dom.development.js:19055:16
 HTMLUnknownElement.callCallback
  node_modules/react-dom/cjs/react-dom.development.js:3945:14
 Object.invokeGuardedCallbackDev
  node_modules/react-dom/cjs/react-dom.development.js:3994:16
 invokeGuardedCallback
  node_modules/react-dom/cjs/react-dom.development.js:4056:31
 beginWork$1
  node_modules/react-dom/cjs/react-dom.development.js:23959:7
```
Which is expected.. because component `EntityPagerDutyCard` seems to be designed for entity pages? On home page, how would that component know about the integration-key/service-id? Hence my original question about the extensibility of the plugin. Please correct me if I am wrong.

**Comment by Rugvip** [comment]
Yep, the `EntityPagerDutyCard` is indeed only for use on entity pages.

Basically what is needed here is to build a new extension for the pager duty plugin that provides a component that can be used on the homepage, which is the original ask.

**Comment by parifuture** [comment]
Ah I see that makes sense, the entity context only exists within catalog. A suggestion, defiantly not a replacement for your ask, we are using `Starred Entities` plugin which allows anyone to add Entites that they care about and then they can go in to see any specific onCall/PD Incidents details.
